packages/react-components/src/shared/src/useFocusManager.ts

```diff
@@ -2,11 +2,11 @@ import { FocusTarget } from "./focusTarget";
 import { isFunction, isNil } from "lodash";
 import { useMemo } from "react";
 
-class ElementIterator {
+class ElementIterator<T> {
```

Comment:
I don't think you need to make this generic. This iterator is strictly for HtmlElement

Reply:
if you insist i could. However, since this class work for any array of element, i don't see why we would limit ourselves with that